java/sqlcommenter-java/src/test/java/com/google/cloud/sqlcommenter/schibernate/SCHibernateTest.java

```diff
@@ -17,25 +17,29 @@
 import static com.google.common.truth.Truth.assertThat;
 
 import com.google.cloud.sqlcommenter.threadlocalstorage.State;
-import io.opencensus.common.Scope;
-import io.opencensus.trace.SpanContext;
-import io.opencensus.trace.Tracer;
-import io.opencensus.trace.Tracing;
+
 import io.opencensus.trace.samplers.Samplers;
-import org.junit.After;
-import org.junit.Before;
-import org.junit.Test;
+import io.opentelemetry.sdk.trace.TracerSdkProvider;
+import org.junit.*;
```

Comment:
Wildcard is not recommended to use, please replace with imports of specific classes.
Note that you can configure IntelliJ to disable wildcard imports: https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports